ontotext-yasgui-web-component/src/i18n/locale-en.json

```diff
@@ -99,6 +99,10 @@
   "yasqe.keyboard_shortcuts.dialog.item.esc.description": "Press Esc to exit full screen",
   "yasqe.keyboard_shortcuts.action.explain_plan_query.warning.message": "Explain only works with SELECT, CONSTRUCT or DESCRIBE queries.",
   "yasqe.keyboard_shortcuts.action.explain_plan_query.error.message": "Cannot execute updates from this editor.",
+  "yasqe.close_tab.ran_query.warning.message": "<div class=\"closeTabsWarning\">You have running 1 query{{updateInfo}}. The query will be aborted.</div>",
+  "yasqe.close_tab.ran_queries.warning.message": "<div class=\"closeTabsWarning\">You have running {{queryCount}} queries{{updateInfo}}.The queries will be aborted.</div>",
+  "yasqe.close_tab.ran_update.warning.message": " and 1 update",
```

Comment:
I don't like this method for assembling these labels. I'd rather prefer having four similar messages: two with update part and two without.